src/app/store.ts

```diff
@@ -0,0 +1,16 @@
+import { configureStore } from '@reduxjs/toolkit';
+import todoReducer, { TodoState } from '../features/todo/todoSlice';
+
+export interface RootState {
+  todos: TodoState;
+  // Add other reducers here if needed
+}
+
+export const store = configureStore({
+  reducer: {
+    todos: todoReducer,
```

Comment:
Please rename this reducer to `todosReducer` so it's different than its `todos` state.

Reply:
I meant `todoReducer: todoReducer` here.